.github/workflows/test.yaml

```
@@ -498,7 +538,7 @@ jobs:
 
   release:
     needs: [build, code-quality, test-install, integration-test, integration-test-vue3, unit-test]
-    if: startsWith(github.event.ref, 'refs/tags/v')
+    # if: startsWith(github.event.ref, 'refs/tags/v')
```

I'm guessing the intention is to remove the comment later?

```suggestion
    if: startsWith(github.event.ref, 'refs/tags/v')
```